### DISCUSSION TITLE
Support for biometrics

### DISCUSSION THREAD

**Discussion by johndekroon** [codeigniter4/shield#1071] [Ideas]
It would be really nice if we can support biometrics as 2fa. 
With [webauthn](https://webauthn.io) this is possible. Is an integration possible?

**Comment by datamweb** [comment]
> Is an integration possible?

Shield does not limit you, if you master [webauthn](https://webauthn.io/) you can do this implementation. Refer to [ defining-new-actions](https://shield.codeigniter.com/references/authentication/auth_actions/?h=2fa#defining-new-actions).
